app/src/screens/navigation-entries.screen.tsx

```diff
@@ -30,16 +30,36 @@ import { getBrowserIconFromDevice } from '../utils';
 import clsx from 'clsx';
 
 import { updateIcon } from '../utils/updateIcon';
-
+import Markdown from 'react-markdown';
 const truncateString = (str: string, maxLength: number) => {
   return str.length <= maxLength ? str : str.slice(0, maxLength) + '...';
 };
-
+const getPreProcessedMarkDown = (relevantSegment: string) => {
+  const emptyListPatterns = [
+    /\*\n(\*\n)*/g, // matches lines with only *
+    /-\n(-\n)*/g, // matches lines with only -
+    /- \*\*\n(\*+\n)*/g, // matches lines with only - ** and *+
+    /\n\s*\*\*\n(\s*\*+\n)*/g, // matches lines with ** and *+ with spaces
+    /- \*\n(\s*\*\n)*/g, // matches lines with - * and *+
+    /\n\s*\*\n(\s*\*\n)*/g, // matches lines with * and *+ with spaces
+    /\n\s*-\n(\s*-\n)*/g, // matches lines with - and -+ with spaces
+  ];
+  let markdown = relevantSegment;
+  emptyListPatterns.forEach((pattern) => {
+    markdown = markdown.replace(pattern, '');
+  });
+  markdown = markdown.replace(/\n{2,}/g, '\n\n');
+  return markdown || '';
+};
 const RelevantSegment = ({ relevantSegment }: { relevantSegment: string }) => {
+  const markdown = getPreProcessedMarkDown(relevantSegment);
   return (
     <div>
       <p className='font-semibold mb-4'>Most relevant match</p>
-      <p className='text-xs'>{relevantSegment}</p>
+      <p className='text-xs'></p>
```

Comment:
@agustinferrentino is this empty paragraph needed?

Reply:
Um, really not. I'll fix it